lib/grammers-client/examples/dialogs.rs

```diff
@@ -58,7 +61,19 @@ async fn async_main() -> Result<()> {
         let phone = prompt("Enter your phone number (international format): ")?;
         let token = client.request_login_code(&phone, api_id, &api_hash).await?;
         let code = prompt("Enter the code you received: ")?;
-        client.sign_in(&token, &code).await?;
+        let signed_in = client.sign_in(&token, &code).await;
+        match signed_in {
+            Ok(_) => (),
+            Err(SignInError::PasswordRequired) => {
+                let password = prompt("Enter the password: ")?.to_bytes();
```

Comment:
This is a good showcase on how to sign in when password is required, but we probably want to add a comment saying something along the lines of:

```rust
// Note: this `prompt` method will echo the password in the console.
//       Real code might want to use a better way to handle this.
```